docs/changelog.md

```diff
@@ -35,6 +35,9 @@ permalink: /changelog/
 
 * **browser-tabstray**
   * The iconView is no longer required in the template.
+  
+* **browser-toolbar**
+  * It will only be animated for vertical scrolls inside the EngineView. Not for horizontal scrolls. Not for zoom gestures.
```

Comment:
nit: Can you move this to the latest release? (Sorry, that the review took so long)

Reply:
Sure, np!